h/static/scripts/directive/form-validate.coffee

```diff
@@ -2,15 +2,15 @@ module.exports = ->
   controller: ->
     controls = {}
 
-    addControl: (control) ->
+    this.addControl = (control) ->
```

Comment:
But that said, if this is the case then `controls` should become an instance property rather than a closed-over variable.